telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/TelemetryGenerator.kt

```diff
@@ -40,13 +40,38 @@ fun generateTelemetryFromFiles(
     outputFolder: File
 ) {
     val telemetry = TelemetryParser.parseFiles(defaultDefinitions, inputFiles)
+    val commonMetadataTypes = setOf(
+        "awsAccount",
+        "awsRegion",
+        "duration",
+        "httpStatusCode",
+        "reason",
+        "reasonDesc",
+        "requestId",
+        "requestServiceType",
+        "result",
```

Comment:
would have been nice to include a case to make sure all of these are generated

Reply:
sure, will add them into test inputs and outputs